### ISSUE TITLE
Faktura pro PyData Prague Meetup -- Rapid7

### ISSUE THREAD

**Issue by jiri-bajer** [pyvec/money#28]
**Kdo jste?**
Jirka Bajer, PyData Prague Meetup org

**Na jakou věc to je?**
Listopadovy PyData Prague Meetup v Rapid7

**Proč? (čemu prospějeme, proč je to dobrý nápad, jaký to bude mít dopad?)**
Sponzorsky prispevek od Rapid7

**Částka**
10 000,- Kc

**Fakturační údaje protistrany**
Rapid7 Czech Republic s.r.o.
V celnici 1031/4, 110 00 Praha 1, Czech Republic
ICO: 19325738, DIC: CZ19325738

**E-mail protistrany**
Prosim o poslani odkazu na fakturu ve Fakturoidu do Slack kanalu #money

**Je potřeba podepsat nějakou smlouvu?**
Ne

**Comment by honzajavorek** [comment]
Je toto vyřešeno? To bylo jen otočení sponzorských peněz, že? @baradrb

**Comment by baradrb** [comment]
Jo. Rapid7 potřeboval potvrzení, že jsme majitelem bankovního účtu, aby nás zapsali jako svého vendora. Viz slack thread. Peníze nám přišly 7.2. Fakturu jsem původně smazala z fakturoidu, protože se čekalo na ten jejich proces, tak ji vytvořím znovu.
